package.json

```diff
@@ -4,11 +4,11 @@
   "description": "A library that handles payload encryption for the Web Push protocol",
   "main": "src/index.js",
   "scripts": {
-    "test": "gulp lint test:manual",
+    "publish-release": "./node_modules/sw-testing-helpers/project/publish-release.sh",
```

Comment:
For consistency’s sake: Worth adding a `publish-docs` script ehre here and calling that from `.travis.yml`?